From ee5ef5a7678035e38c28209b1a44c31d36d7610e Mon Sep 17 00:00:00 2001
Message-ID: <ee5ef5a7678035e38c28209b1a44c31d36d7610e.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 01:17:29 +0100
Subject: [PATCH 17/19] gnu: go-golang-org-x-lint: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-golang-org-x-lint): Move from
here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 28 ----------------------------
 2 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 011198231d..56f412224e 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -458,6 +458,34 @@ (define-public go-gopkg-in-go-playground-assert-v1
 custom assertions to be used alongside native Go testing.")
     (license license:expat)))
 
+(define-public go-golang-org-x-lint
+  (let ((commit "83fdc39ff7b56453e3793356bcff3070b9b96445")
+        (revision "0"))
+    (package
+      (name "go-golang-org-x-lint")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://go.googlesource.com/lint")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "0ms3rs5hvpnm9bxbr5f9743i7hn2bbmqdmvzxq6nmi0f24ypv1l3"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "golang.org/x/lint"
+         #:tests? #f)) ;; TODO: Fix tests
+      (propagated-inputs
+       (list go-golang-org-x-tools))
+      (home-page "https://golang.org/x/lint")
+      (synopsis "Linter for Go source code")
+      (description
+       "This is a linter for Go source code.  Unlike gofmt, it doesn't
+reformat the source code, it only prints out style mistakes.")
+      (license license:bsd-3))))
+
 ;;;
 ;;; Avoid adding new packages to the end of this file. To reduce the chances
 ;;; of a merge conflict, place them above by existing packages with similar
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 2503fca0b6..e5dd116766 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11477,34 +11477,6 @@ (define-public go-mvdan-cc-xurls
     (native-inputs '())
     (inputs '())))
 
-(define-public go-golang-org-x-lint
-  (let ((commit "83fdc39ff7b56453e3793356bcff3070b9b96445")
-        (revision "0"))
-    (package
-      (name "go-golang-org-x-lint")
-      (version (git-version "0.0.0" revision commit))
-      (source
-       (origin
-         (method git-fetch)
-         (uri (git-reference
-               (url "https://go.googlesource.com/lint")
-               (commit commit)))
-         (file-name (git-file-name name version))
-         (sha256
-          (base32 "0ms3rs5hvpnm9bxbr5f9743i7hn2bbmqdmvzxq6nmi0f24ypv1l3"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "golang.org/x/lint"
-         #:tests? #f)) ;; TODO: Fix tests
-      (propagated-inputs
-       (list go-golang-org-x-tools))
-      (home-page "https://golang.org/x/lint")
-      (synopsis "Linter for Go source code")
-      (description
-       "This is a linter for Go source code.  Unlike gofmt, it doesn't
-reformat the source code, it only prints out style mistakes.")
-      (license license:bsd-3))))
-
 (define-public go-github-com-kisielk-gotool
   (package
     (name "go-github-com-kisielk-gotool")
-- 
2.41.0

